src/Uno.UI/Microsoft/UI/Xaml/Media/RadialGradientBrush.Android.cs

```diff
@@ -40,10 +38,7 @@ protected internal override Shader GetShader(Rect destinationRect)
 			var colors = GradientStops.SelectToArray(s => ((Android.Graphics.Color)GetColorWithOpacity(s.Color)).ToArgb());
 			var locations = GradientStops.SelectToArray(s => (float)s.Offset);
 
-			var width = destinationRect.Width;
-			var height = destinationRect.Height;
-
-			var transform = RelativeTransform?.ToNativeMatrix(size: new Windows.Foundation.Size(width, height));
+			var transform = RelativeTransform?.ToNativeMatrix(size: size);
```

Comment:
Remove if not needed 👍

Reply:
@MartinZikmund I'm not sure if it's needed or not. This unused variable was already unused before my changes.

I don't know, but isn't it possible the method call has an *intended* side-effect?, e.g:

https://github.com/unoplatform/uno/blob/b3d691f128881b484265dbdc116b9e2029731ffb/src/Uno.UI/UI/Xaml/Media/Transform.Android.cs#L24-L29